### ISSUE TITLE
connected to com.apple.mobile.restored

### ISSUE THREAD

**Issue by kloytofyexploiter** [CoocooFroggy/FutureRestore-GUI#95]
Pls someone Help me i Get this error connected in Restore mode
Restore_is_current_device connected to com-apple.mobile.restored version 15

**Comment by CoocooFroggy** [comment]
Have you tried turning on "Preview command" in settings, then pressing "Start FutureRestore" > "Copy Command Only"? Then you can paste this into the terminal to run FutureRestore and see if it gets you any more info.

**Comment by TheRazerMD** [comment]
@CoocooFroggy I had the same issue as @kloytofyexploiter on my iPad Air 2. Then I ran it with the Preview Command and through Terminal and got more info. Error was "Segmentation fault (core dumped)". After some very helpful people on Discord told me to use an older version of FutureRestore Beta, I tried the first older one from 3 months ago (March 2022).

And presto, it worked great! Seems like something is broken with the latest FR Beta from a week ago. 

Also if anyone is reading this trying to FutureRestore through Linux/Ubuntu Live USB

Don't use Ubuntu 22 LTS, use Ubuntu 20 LTS so the linux_fix.sh script (for fixing USB not reading iDevices in Recovery mode) doesn't break your system for some reason (atleast it did on mine). 
Also use FutureRestore Beta from 3 months ago (March 2022).
